packages/palette/src/elements/Button/Button.tsx

```diff
@@ -28,13 +28,26 @@ export class Button extends Component<ButtonProps> {
   }
 
   getSize(): { height: string; size: "2" | "3t"; px: number } {
+    const { inline } = this.props
     switch (this.props.size) {
       case "small":
-        return { height: "26px", size: "2", px: 2 }
+        return {
+          height: inline ? "17px" : "26px",
+          size: "2",
+          px: inline ? 0 : 1,
```

Comment:
This changes the original behavior to have 10px padding instead of 20px. Is that intended?

Reply:
This also shows up in the chromatic test.

Reply:
This only applies to `<Button inline />` I just checked reaction for its usage and there are no instances of it as of now.

Reply:
@samchieng just to double check, does this change sound ok?